packages/cli/src/commands/push.ts

```diff
@@ -224,17 +224,15 @@ function getDestinationProps(destination: string) {
   return destination.substring(1).split(/[@\/]/);
 }
 
-async function uploadFileToS3(url: string, filePathOrBuffer: string | Buffer): Promise<boolean> {
+function uploadFileToS3(url: string, filePathOrBuffer: string | Buffer) {
```

Comment:
why did you remove async?

Reply:
because I removed await inside the function